packages/dx-react-scheduler-material-ui/src/templates/edit-recurrence-menu/layout.jsx

```diff
@@ -1,28 +1,36 @@
 import * as React from 'react';
+import { styled } from '@mui/material/styles';
 import * as PropTypes from 'prop-types';
 import DialogActions from '@mui/material/DialogActions';
 import DialogContent from '@mui/material/DialogContent';
 import DialogTitle from '@mui/material/DialogTitle';
 import Radio from '@mui/material/Radio';
 import RadioGroup from '@mui/material/RadioGroup';
 import FormControlLabel from '@mui/material/FormControlLabel';
-import withStyles from '@mui/styles/withStyles';
 import { SMALL_LAYOUT_MEDIA_QUERY } from '../constants';
 
-const styles = ({ typography }) => ({
-  title: typography.h6,
-  content: {
+const PREFIX = 'Layout';
+
+export const classes = {
+  title: `${PREFIX}-title`,
+  content: `${PREFIX}-content`,
+  media: `${PREFIX}-${SMALL_LAYOUT_MEDIA_QUERY}`,
+};
+
+const StyledDiv = styled('div')(({ theme: { typography } }) => ({
+  [`& .${classes.title}`]: typography.h6,
```

Comment:
should not be the spacing

Reply:
This style used in children component, space need